crates/verify/src/etherscan/flatten.rs

```diff
@@ -70,7 +70,7 @@ impl EtherscanSourceProvider for EtherscanFlattenedSource {
         args: &VerifyArgs,
         context: &ZkVerificationContext,
     ) -> Result<(String, String, CodeFormat)> {
-        let metadata = context.project.settings.metadata.as_ref();
+        let metadata = context.project.settings.settings.metadata.as_ref();
```

Comment:
Can we not rename `settings.settings` ?

Reply:
We can but `foundry-compilers` does this with `Solc` so it is less cognitive overload to port things if stuff is named the same.

Reply:
`json_settings`  could be a good name for `ZkSettings` though